soc/xtensa/esp32/linker.ld

```diff
@@ -230,7 +230,7 @@ SECTIONS
 
   .dram0.data :
   {
-    _data_start = ABSOLUTE(.);
+    __data_start = ABSOLUTE(.);
 
```

Comment:
This change needs to be applied to esp32_net as well.

Reply:
But I've already did that at https://github.com/zephyrproject-rtos/zephyr/pull/51466/files#diff-2f766f557e3b4c2487a0529179adf7b1848721e67ad25ae848035a36fa514367
Maybe you are thinking of some other place?